ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -216,6 +260,54 @@ export default class StatusDropdown extends React.Component {
     alertStatus === 'investigating' ||
     (alertStatus !== status && this.isResolved(alertStatus));
 
+  calculateDueDate(created) {
+    const createdAt = new Date(created);
+    const dueDate = new Date(created);
+    dueDate.setDate(dueDate.getDate() + 3);
```

Comment:
I'd move the `3` into TIME_TO_TRIAGE global and add a function that calculates the due date depending on it, so you easily add `TIME_TO_BUG = 5`.
BTW, the terms are alertToTriage and alertToBug. probably worth syncing the naming, but it's your choice.